util/cron/test-ml-memleaks.bash

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+#
+# Test gasnet (segment everything) against full suite on linux64.
+
+CWD=$(cd $(dirname $0) ; pwd)
+source $CWD/common-gasnet.bash
+source $CWD/common-localnode-paratest.bash
+source $CWD/common-memleaks.bash
+
+export CHPL_NIGHTLY_TEST_CONFIG_NAME="ml-memleaks"
+
+export GASNET_QUIET=Y
+
+# Test a GASNet compile using the default segment (everything for linux64)
+export CHPL_GASNET_SEGMENT=everything
+
+$CWD/nightly -cron -futures -multilocale -memleaks $(memleaks_log ml) $(get_nightly_paratest_args 3)
```

Comment:
Let's avoid oversubscribing with paratest since this will run on a machine that's not very beefy

Remove `$(get_nightly_paratest_args 3)` and `source $CWD/common-localnode-paratest.bash`

Reply:
`common-localnode-paratest.bash` was already sourced. Removed the call to the argument generating script

Reply:
Yeah, I meant sourcing it can be removed too (since it's used to enable parstest, and we're not using paratest anymore)

Reply:
ugh.. done. 

I'll squash-merge this PR, I don't think commits are valuable